### ISSUE TITLE
Trade XP stop levelling up

### ISSUE THREAD

**Issue by Seralph5** [Tyler-IN/MnB2-Bannerlord-CommunityPatch#166]
No XP gained when making a profitable trade.

**Comment by Skau** [comment]
AFAIK this is because the game does not save what price you bought items at. If you sell something profitable during the same game as you bought it, you should gain XP as normal. If you load a game the buying price of items bought will not be present, and no XP will be added.

**Comment by Solganis** [comment]
Yep, it's pretty annoying. Can't level trade at any reasonably rate.

**Comment by SteelBlood2** [comment]
There is a fix for this: https://www.nexusmods.com/mountandblade2bannerlord/mods/747

**Comment by Solganis** [comment]
For me it doesn't fix it's problem. 1.2 Beta branch.

**Comment by Tyler-IN** [comment]
Is this something we need to address too?

**Comment by Solganis** [comment]
@Tyler-IN I'm not sure, but they fixed it in 1.4 Beta
Need to re-read the patch notes for 1.3 stable release and 1.4 Beta
